lib/kubeclient/common.rb

```diff
@@ -216,7 +216,11 @@ def create_entity(entity_type, entity_config, klass)
       # TODO: temporary solution to add "kind" and apiVersion to request
       # until this issue is solved
       # https://github.com/GoogleCloudPlatform/kubernetes/issues/6439
-      hash[:kind]       = entity_type
+      if entity_type.eql? 'Endpoint'
```

Comment:
> @abonas do we need also another comment with the reference to this specific issue? (Remind me... is the issue on their side still open?)

I'll add a comment, it doesn't cost money :)
yes, the issue on server side is still open.